dev-libs/botan/botan-2.0.1.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A cryptography library written in C++"
+DESCRIPTION="This is the best option for crypthography in C++"
+HOMEPAGE="https://botan.randombit.net/"
+COPYRIGHT="2000-2017 botan"
```

Comment:
Well, at least in France you cannot give your authorship away. You can give the copyright, but as soon as you change a line of code, you are part of the "authors".

And yes, we should squeeze the date range, of course.